package.json

```diff
@@ -61,7 +61,10 @@
     "@justeat/pie-design-tokens": "4.2.0",
     "@justeat/stylelint-config-fozzie": "3.x",
     "@percy/cli": "1.20.1",
+    "@percy/playwright": "1.0.4",
     "@percy/webdriverio": "2.0.2",
+    "@playwright/test": "1.31.2",
+    "@sand4rt/experimental-ct-web": "1.0.7",
     "@wdio/allure-reporter": "7.26.0",
     "@wdio/cli": "7.27.0",
     "@wdio/devtools-service": "7.27.0",
```

Comment:
why do we still need these? If it's in case we need to revert then we can rely on git for that

Reply:
We still need these for a11y / visual until they're refactored 👍🏼